CMakeLists.txt

```
@@ -680,6 +680,24 @@ install(TARGETS isle ${install_extra_targets}
   LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
 )
 if (ISLE_BUILD_CONFIG)
+  find_program(WINDEPLOYQT_EXECUTABLE windeployqt)
+  if(WINDEPLOYQT_EXECUTABLE)
+    add_custom_command(TARGET isle-config POST_BUILD
+      COMMAND "${WINDEPLOYQT_EXECUTABLE}"
+              "$<TARGET_FILE:isle-config>"
+              "--dir"
+              "QTLibs"
+              "--no-compiler-runtime"
+              "--no-opengl-sw"
+              "--no-system-d3d-compiler"
+              "--no-translations"
+              "--no-quick-import"
+      COMMENT "Running windeployqt with minimal dependencies"
+    )
+    install(DIRECTORY "Build/QTLibs/" DESTINATION "${CMAKE_INSTALL_BINDIR}")
+  else()
+    message(WARNING "windeployqt not found.")
```

A warning is too harsh imho, and we want to make it possible to install `isle-config` without qt binaries (e.g. msys2)
```suggestion
    message(STATUS "windeployqt not found: Qt binaries will not be installed")
```